plugin/src/main/java/com/denizenscript/denizen/scripts/commands/entity/FakeInternalDataCommand.java

```diff
@@ -0,0 +1,98 @@
+package com.denizenscript.denizen.scripts.commands.entity;
+
+import com.denizenscript.denizen.nms.NMSHandler;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.Utilities;
+import com.denizenscript.denizencore.DenizenCore;
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsRuntimeException;
+import com.denizenscript.denizencore.objects.core.DurationTag;
+import com.denizenscript.denizencore.objects.core.MapTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.denizencore.scripts.commands.generator.*;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Player;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.LockSupport;
+
+public class FakeInternalDataCommand extends AbstractCommand {
+
+    public FakeInternalDataCommand() {
+        setName("fakeinternaldata");
+        setSyntax("fakeinternaldata [entity:<entity>] [data:<map>|...] (for:<player>|...) (speed:<duration>)");
+        setRequiredArguments(2, 4);
+        autoCompile();
+    }
+
+    // <--[command]
+    // @Name FakeInternalData
+    // @Syntax fakeinternaldata [entity:<entity>] [data:<map>|...] (for:<player>|...) (speed:<duration>)
+    // @Required 3
+    // @Maximum 4
+    // @Short Sends fake entity data updates, optionally animating them with sub-tick precision.
+    // @Group entity
+    //
+    // @Description
+    // Sends fake internal entity data updates, optionally sending multiple over time.
+    // This supports sub-tick precision, allowing smooth/high FPS animations.
+    //
+    // The input to 'data:' is a list of <@link object MapTag>s, with each map being a frame to send, with each map being formatted like <@link mechanism EntityTag.internal_data>'s input.
```

Comment:
A command should probably be considered "more primary" than a mech (ie put the details in command and have the mech link to it) - or add a separate `language` defining the data that both link to

Reply:
Pushed an initial attempt at a language doc for this, let me know if it looks good